### DISCUSSION TITLE
article/spring-boot/spring-boot-stop-use-value

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#26] [Announcements]
# article/spring-boot/spring-boot-stop-use-value

为什么不推荐在Spring Boot中使用@Value加载配置 @Value注解相信很多Spring Boot的开发者都已经有接触了，通过使用该注解，我们可以快速的把配置信息加载到Spring的Bean中。 比如下面这样，就可以轻松的把配置文件中key为com.didispace.title配置信息加载到TestService中来使用 @Service public class TestService { @Value(

https://www.didispace.com/article/spring-boot/spring-boot-stop-use-value.html

**Comment by luoshupeng** [comment]
这个真正学到了

**Comment by ShiftWatchOut** [comment]
而且我日常使用中`@Value`注解所在字段好像只能是简单类型，如果是集合或者复合类型就没法启动项目，也不知道是不是我用得不对
